src/tensorlake/applications/interface/image.py

```diff
@@ -87,6 +130,54 @@ def run(
             )
         )
 
+    def pip_install(self, packages: str | List[str]) -> "Image":
+        """
+        Install Python packages using pip in the image. Equivalent to running 'pip install' in a RUN command.
+
+        Args:
+            packages (str | List[str]): The package or list of packages to install.
+        """
+
+        pkgs = packages if isinstance(packages, list) else [packages]
+        install_cmd = f"pip install {' '.join(pkgs)}"
+        return self.run(install_cmd)
+
+    def cmd(self, command: str | List[str]) -> "Image":
```

Comment:
Not great that we're adding OSS only functionality to SDK. Can we move this to Indexify as an extended image functionality?